tls/ttls.c

```diff
@@ -2151,7 +2151,9 @@ ttls_recv(void *tls_data, unsigned char *buf, unsigned int len, unsigned int *re
 		unsigned int delta;
 
 		if ((r = ttls_parse_record_hdr(tls, buf, len, read))) {
-			TTLS_WARN(tls, "Bad TLS record (error -0x%X)\n", r);
+			if (unlikely(r != T_POSTPONE))
+				TTLS_WARN(tls,
+					  "Bad TLS record (err -0x%X)\n", r);
```

Comment:
If we place this on single line, then it's just 82, not so huge violation from 80